community/blog/2020.10-Introducing-message-handling-with-Azure-Service-Bus-message-influence/README.md

```diff
@@ -0,0 +1,107 @@
+# Introducing message handling with Azure Service Bus message influence in v0.5
+
+Starting from v0.4 and updated in v0.5, the [Arcus.Messaging]() has got an upgrade in handling messages from Azure Service Bus.
```

Comment:
```suggestion
Starting from v0.4 and updated in v0.5, the [Arcus.Messaging](https://github.com/arcus-azure/arcus.messaging) has got an upgrade in handling messages from Azure Service Bus.
```

Reply:
Yeah, was planning to add it. 😄 . Thanks.